### ISSUE TITLE
ERROR An invalid version was provided.

### ISSUE THREAD

**Issue by jgentes** [webpro/release-it#431]
If no release type is provided (ie `npm run release`), I get:

```
🚀 Let's release @tend/boss (currently at 5.5.0)
ERROR An invalid version was provided.
```

If I use `npm run release patch` or `npm run release minor` it works fine.

Was the default 'patch' removed in v9?

**Comment by jgentes** [comment]
Hmm, ok I'll need to take a look. `npm run release minor` works fine, without the -- prefix.

Here's our config:

```
{
  "non-interactive": true,
  "git": {
    "requireCleanWorkingDir": false,
    "tagName": "v${version}"
  },
  "npm": {
    "publish": false
  }
}
```

**Comment by webpro** [comment]
You're right about not needing the `--`. Usually it's necessary, even the [npm run-script docs](https://docs.npmjs.com/cli/run-script.html) say so.

I could reproduce the issue, and a fix has landed in [v9.4.2](https://github.com/webpro/release-it/releases/tag/9.4.2). Thanks for reporting this!
